content/docs/user-guide/components/reference/physx/collider.md

```diff
@@ -1,258 +1,148 @@
+## Properties 
 
-**Offset**
-Local offset position of the collider, relative to the entity.
+![PhysX Collider component interface.](/images/user-guide/components/reference/physx/physx-collider-ui-01.png)
 
-**Rotation**
-Local rotation of the collider about the **Offset** of the PhysX collider component.
+### Base properties
 
-**Library (Physics Materials)**
-The physics material library of the project.
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Collision Layer** | Assigns the collider to a collision layer. Collision layers can be used to restrict physical interactions between PhysX objects. | Any collision layer defined the project [Collision Layers](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers/).  | `Default` |
+| **Collides With** | Assigns the collider to a collision group. Collision groups contain the collision layers that this collider can collide with. | Any collision group defined the project [Collision Groups](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-groups/). | `All` |
+| **Trigger** | If enabled, this collider functions as a trigger. A trigger performs a quick overlap test and does not apply forces or return contact point information. Use this to speed-up PhysX computations where a simple overlap test between colliders is sufficient. Triangle meshes are not supported as triggers. | Boolean | `Disabled` |
+| **Simulated** | If enabled, this collider is included in the physics simulation. | Boolean | `Enabled` |
+| **In Scene Queries** | If enabled, this collider can be queried for raycasts, shapecasts, and overlap. | Boolean | `Enabled` |
+| **Offset** | Sets a local offset position for the collider relative to the entity. | Vector3: -Infinity to Infinity | X: `0.0`, Y: `0.0`, Z: `0.0` |
+| **Rotation** | Sets a local rotation for the collider around the **Offset** of the PhysX collider component. | Vector3: -180 to 180 | X: `0.0`, Y: `0.0`, Z: `0.0` |
+| **Library (Physics Materials)** | The physics material library of the project. | A `.physmaterial` library product asset. | The global project `.physmaterial` library. |
+| **Slots (Physics Materials)** | Choose a physics material for each material of this collider. Physics materials define physical properties for the surface such as dynamic and static friction, and density. A collider can have multiple physics materials assigned. | Physics material(s) from the assigned `.physmaterial` library | `<Default Physics Material>` |
+| **Tag** | Sets a tag for this collider. Tags can be used to quickly identify components in script or code. | String |  |
+| **Rest offset** | Sets the minimum distance between this collider and other colliders when this collider is at rest. PhysX bodies come to rest separated by the sum of their **Rest offset** values. The **Rest offset** value must be less than the **Contact offset** value. | Float: -Infinity to 50 | `0.0` |
```

Comment:
Building off of FiniteStateGit's suggestion.
I suggest adding "position" to specify what's being separated. Alternatively, you can say: "PhysX bodies come to rest at a position separated by..."
```suggestion
| **Rest offset** | Sets the minimum distance between this collider and other colliders when this collider is at rest. PhysX bodies's rest positions are separated by the sum of their **Rest offset** values. The **Rest offset** value must be less than the **Contact offset** value. | Float: -Infinity to 50.0 | `0.0` |
```

Reply:
I don't think the term rest position should be used here because it implies a specific default/start position for an animation or simulation. This is the body coming to rest in some undefined position due to friction/damping.

Reply:
How about: "PhysX bodies come to rest at a position separated by the sum of their **Rest offset** values."

I think it helps to clarify _what_ is "separated by the sum of their **Rest offset** values"

Reply:
Thinking about this, this is actually a poorly named property. Technically, the concept of rest really only applies to dynamic colliders, but every collider type uses this offset. Maybe this is better. I really want to avoid using the terms 'rest' and 'position' in close proximity in this context because of what it implies.

```suggestion
| **Rest offset** | Sets the minimum distance between this collider and other colliders. Although this property applies to all colliders, it is particularly important for dynamic colliders. Dynamic colliders are at rest when the forces affecting them drop below the **Sleep threshold** of their rigid body component. When a dynamic collider comes to rest while in contact with any other collider, the colliders are separated by the sum of their **Rest offset** values. **Rest offset** values that are too large might make dynamic entities appear to float. Negative **Rest offset** values might make dynamic entities appear to intersect other entities on collision. You might need to adjust this value in scenarios where the collider does not closely match the render mesh of the entity to improve the visual fidelity of PhysX simulations. The **Rest offset** value must be less than the **Contact offset** value. | Float: -Infinity to 50.0 | `0.0` |
```